parlai/tasks/ubuntu/agents.py

```diff
@@ -113,70 +113,5 @@ def share(self):
         return shared
 
 
-class UbuntuTeacherWithNegs(UbuntuTeacher):
```

Comment:
This is not needed anymore?

Reply:
Untested and didn’t work. So kill it lol